docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,3685 @@
+(https://keras.io/datasets/#imdb-movie-reviews-sentiment-classification).
+The model has one output node that gives probabilities. Output close to
+1 is positive, and close to 0 is negative.
+
+See
+https://www.tensorflow.org/beta/tutorials/text/text\_classification\_rnn
+for a simple example of how to build such a model and prepare its input.
+
+Let's load our pre-trained model
+
+.. code:: ipython3
+
+    binary_model = keras.models.load_model('tests/estimators/keras_sentiment_classifier/keras_sentiment_classifier.h5')
+    binary_model.summary()
+
+
+.. parsed-literal::
+
+    _________________________________________________________________
+    Layer (type)                 Output Shape              Param #   
+    =================================================================
+    embedding_1 (Embedding)      (None, None, 8)           80000     
+    _________________________________________________________________
+    masking_1 (Masking)          (None, None, 8)           0         
+    _________________________________________________________________
+    masking_2 (Masking)          (None, None, 8)           0         
+    _________________________________________________________________
+    masking_3 (Masking)          (None, None, 8)           0         
+    _________________________________________________________________
+    bidirectional_1 (Bidirection (None, None, 128)         37376     
+    _________________________________________________________________
+    bidirectional_2 (Bidirection (None, None, 64)          41216     
+    _________________________________________________________________
+    bidirectional_3 (Bidirection (None, 32)                10368     
+    _________________________________________________________________
+    dense_1 (Dense)              (None, 8)                 264       
+    _________________________________________________________________
+    dense_2 (Dense)              (None, 1)                 9         
+    =================================================================
+    Total params: 169,233
+    Trainable params: 169,233
+    Non-trainable params: 0
+    _________________________________________________________________
+
+
+Load our test and train data. We have a module that will do
+preprocessing for us. For your own usage you may have to do your own
+preprocessing.
+
+.. code:: ipython3
+
+    import tests.estimators.keras_sentiment_classifier.keras_sentiment_classifier \
+    as keras_sentiment_classifier
+
+.. code:: ipython3
+
+    (x_train, y_train), (x_test, y_test) = keras_sentiment_classifier.prepare_train_test_dataset()
+
+Confirm the accuracy of the model
+
+.. code:: ipython3
+
+    print(binary_model.metrics_names)
+    binary_model.evaluate(x_test, y_test)
+
+
+.. parsed-literal::
+
+    ['loss', 'acc']
+    25000/25000 [==============================] - 88s 4ms/step
+
+
+
+
+.. parsed-literal::
+
+    [0.4319177031707764, 0.81504]
```

Comment:
If there is some way to make it more clear that first value is loss and second is accuracy (or leave only accuracy), that would be nice to have.